WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
Traceback (most recent call last):
  File "/home/jdbl/core/Debloat.py", line 80, in depClean
    output = subprocess.check_output(cmd, shell=True)
  File "/usr/lib/python3.7/subprocess.py", line 395, in check_output
    **kwargs).stdout
  File "/usr/lib/python3.7/subprocess.py", line 487, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command 'cd /tmp/tmp_496lnr8/jide-oss; mvn -gs /home/jdbl/core/../maven-proxy.xml se.kth.castor:depclean-maven-plugin:1.1.0:depclean -Dcreate.pom.debloated=true -Dcreate.result.json=true > /home/jdbl/results/jidesoft_jide-oss/27dc1cdff7e73a00e217b1cd41db75bd10e41751/depClean.log 2>&1' returned non-zero exit status 1.
[ERROR] Command 'cd /tmp/tmp_496lnr8/jide-oss; mvn -gs /home/jdbl/core/../maven-proxy.xml se.kth.castor:depclean-maven-plugin:1.1.0:depclean -Dcreate.pom.debloated=true -Dcreate.result.json=true > /home/jdbl/results/jidesoft_jide-oss/27dc1cdff7e73a00e217b1cd41db75bd10e41751/depClean.log 2>&1' returned non-zero exit status 1.
com.jidesoft:jide-oss:jar:3.7.7-pre

